finetune/base_models/gpt/encoder.py

```diff
@@ -138,6 +138,7 @@ def _encode(self, texts, labels=None):
             if labels is not None:
                 label = labels[i]
             raw_text = text.lower()
+            
             # Only fine to apply this fix because it preserves character locations
             ftfy_text = uncurl_quotes(raw_text)
```

Comment:
Probably don't need this called twice now???

Reply:
It's called on two subtly different places.  It's still needed.